features-json/font-variant-east-asian.json

```diff
@@ -0,0 +1,307 @@
+{
+  "title":"CSS font-variant-east-asian ",
+  "description":"Controls the usage of alternate glyphs for East Asian scripts, like Japanese and Chinese.",
+  "spec":"https://www.digitalpreservation.gov/formats/fdd/fdd000114.shtml",
```

Comment:
Spec URL doesn't seem to be correct...

Reply:
Should be https://drafts.csswg.org/css-fonts-3/#propdef-font-variant-east-asian perhaps?